diff --git a/include/meson.build b/include/meson.build
index 35d213d..0d60778 100644
--- a/include/meson.build
+++ b/include/meson.build
@@ -1 +1 @@
-#install_headers('ent.h')   # XX uncomment to install locally
+install_headers('ent.h')
diff --git a/src/meson.build b/src/meson.build
index 04fe34f..5380c34 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -54,6 +54,6 @@ ent_sources = ['ent.c', config_file]
 libent = static_library('ent', ent_sources,
                         include_directories: inc,
                         c_args: ['-Wall', '-pedantic'],
-                        install: false)   # XX set to true to install locally
+                        install: true)
 
 libent_dep = declare_dependency(include_directories: inc, link_with: libent)
